network-api/networkapi/wagtailpages/models.py

```diff
@@ -14,9 +14,13 @@
     PrimaryPage,
     Styleguide,
 )
+
+# Blog models:
 from .pagemodels.blog.blog import BlogAuthors, BlogPage
 from .pagemodels.blog.blog_index import BlogIndexPage
 from .pagemodels.blog.blog_topic import BlogPageTopic
```

Comment:
I think keeping the adopted module import style is a good idea so these should be updated
It does help readability in this codebase in particular
```
from .pagemodels.blog import blog as blog_models  

blog_models .BlogAuthors 
blog_models .BlogPage
```

Reply:
We are re-importing all the models here to make sure that Django picks those up as models from `wagtailpages`. I think it pays off to be explicit here. I'm also keeping up the current style that is in place. If we were to change it here, I think that would need to be raised as a separate ticket.